lib/auth-record-sync.js

```diff
@@ -198,7 +198,7 @@ function create(alephRecordService, alephFindService, options) {
 
 function hasSubfield(code, value) {
   return (field) => {
-    return field.subfields.some(subfield => subfield.code === code && subfield.value === value);
+    return field.subfields.some(subfield => subfield.code === code && (subfield.value === value || subfield.value === value+"," || subfield.value === value+"."));
```

Comment:
Since this oneliner's getting mightly long, I would change the variable name `subfield` to `s` for example.